### PR TITLE
year month week buttons

### PR DESCRIPTION
**Pull request by mwernimont**

### CODE REVIEW

File: layout/css/internalAnalytics.css [hunk 1]
```diff
@@ -36,6 +36,8 @@ h6{
   min-width:250px;
 	max-width:300px;
 	border-right:2px solid rgb(220,220,220);
+	position:relative;
+	z-index:9000;
```
Comment (jiwalker-usgs) [inline layout/css/internalAnalytics.css]: Is z-index an issue on this page?

Comment (mwernimont) [inline layout/css/internalAnalytics.css]: It was at one point, but not anymore

**Comment by mwernimont** [comment]
As for the hooking up, I would say up to you, but if you're looking for me to do it, I would say JS.

**Comment by mwernimont** [comment]
JR is the one with the big idea of these buttons, so maybe he has ideas of how to hook them up.

**Pull request merged**